js/everything.js

```diff
@@ -402,6 +410,28 @@ function loadTime() {
     }
 }
 
+// Key to the persisted volume level entry in localStorage
+var LOCAL_STORAGE_VOLUME_KEY = "VOLUME_LEVEL";
+
+// The delay to debounce writes of the volume level to localStorage
+var VOLUME_PERSIST_DELAY = 200;
```

Comment:
I meant to bring that up as a matter for discussion, I don't really know what an appropriate number would be. The tradeoff is between potentially doing a lot of disk writes if a user adjusts the volume every few seconds, and debouncing by a much larger delay and potentially losing the value if the user navigates away or their browser crashes. 200ms is just in the range of numbers we use for this kind of thing at my work, but usually the data being persisted there is more critical.